src/com/hms_networks/americas/sc/mqtt/MqttManager.java

```diff
@@ -25,6 +26,15 @@
   /** Boolean flag to control the MQTT thread loop. */
   private boolean mqttThreadRun = true;
 
+  /**
+   * Boolean flag to enable UTF8 string conversion. Will enable non-ascii characters, but decreases
+   * performance by a significant amount. Defaults to false, and can be set by constructor.
+   */
+  private boolean mqttUtf8Convert = false;
+
+  /** Encoding format for strings */
+  private final String UTF_8 = "UTF-8";
```

Comment:
```suggestion
  /** Encoding format for strings */
  private final String STRING_ENCODING_FORMAT = "UTF-8";
```
```suggestion
  /** UTF-8 encoding format for strings */
  private final String UTF_8 = "UTF-8";
```
This might be more of a personal preference, but I feel like the current Javadoc comment doesn't entirely line up with what the variable name is, and one of the above would more clearly align the comment and variable name.